src/link/object.c

```diff
@@ -518,9 +518,11 @@ void obj_ReadFile(char const *fileName, unsigned int fileID)
 
 	tryReadlong(revNum, file, "%s: Cannot read revision number: %s",
 		    fileName);
-	if (revNum != RGBDS_OBJECT_REV)
-		errx("%s is a revision 0x%04" PRIx32 " object file; only 0x%04x is supported",
-		     fileName, revNum, RGBDS_OBJECT_REV);
+	if (revNum != RGBDS_OBJECT_REV) {
+		const char *oldTool = revNum < RGBDS_OBJECT_REV ? "rgbasm" : "rgblink";
```

Comment:
To satisfy checkpatch:
```suggestion
		const char *oldTool = revNum < RGBDS_OBJECT_REV ? "rgbasm" : "rgblink";
	
```

Reply:
Thank you. This is fixed locally but I'm not pushing until I can figure out why regression tests are failing

Reply:
I think you forgot to `#include "version.h"`?

Reply:
I know. See above comment.

Reply:
Yes, and I think this is why regression tests are failing.

Reply:
it's not